llap-server/src/test/org/apache/hadoop/hive/llap/cache/TestCacheContentsTracker.java

```diff
@@ -124,7 +125,7 @@ public void testCacheTagComparison() {
 
   @Test
   public void testEncodingDecoding() throws Exception {
-    Map<String, String> partDescs = new HashMap<>();
+    LinkedHashMap<String, String> partDescs = new LinkedHashMap<>();
```

Comment:
Could you please explain why we need LinkedHashMap?

Reply:
That's because (nested) partition specification (e.g. p1=v11,p2=v12) has to be sorted.

Reply:
Got it! Thanks!